packages/forms-web-app/__tests__/unit/routes/lpa/confirm-remove-user.test.js

```diff
@@ -0,0 +1,26 @@
+const { get, post } = require('../router-mock');
+const confirmAddUserController = require('../../../../src/controllers/lpa-dashboard/confirm-remove-user');
+
+jest.mock('../../../../src/controllers/lpa-dashboard/confirm-remove-user');
+
+describe('routes/lpa-dashboard/confirm-remove-user', () => {
+	beforeEach(() => {
+		// eslint-disable-next-line global-require
+		require('../../../../src/routes/lpa-dashboard/confirm-remove-user');
+	});
+
+	afterEach(() => {
+		jest.resetAllMocks();
+	});
+
+	it('should define the expected routes', () => {
+		expect(get).toHaveBeenCalledWith(
+			'/confirm-remove-user/:id',
```

Comment:
I mentioned this on another PR earlier, and I guess this was the point raised re the value of testing the routes. Personally I think there is very limited value in testing this as we're basically just testing the express router. Not a blocker to passing review, just a talking point.

Reply:
I agree and is one of the things I had mentioned on our dev standards call. 
Happy to remove once we decide that's what we're doing